dialog-db/starting-dialog.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Dialog - Personal Substrate for Cooperative Computing
+date: 2025-08-12
+---
+
+# Dialog - Personal Substrate for Cooperative Computing
+
+You're planning dinner for friends this weekend. Your calendar knows who's coming. Your contacts know Sarah's gluten-free, Tom's vegetarian. As you open a meal planner, it suggests recipes everyone can enjoy, portions adjusted for your group. Your shopping list populates with needed ingredients.
+
+No configurations. No authorizations. No information retrieval from corporate silos. Your tools cooperate through a shared substrate—each deriving insights and contributing new understanding. They complement each other, becoming true [bicycles for your mind][].
+
+## Means of Cooperation
+
+Today's tools cooperate through following coordination models:
+
+**Corporate suites** (Google Workspace, Microsoft Office) work together because one company controls everything.
+
+**Limited integrations** where companies choose which APIs to expose. Slack decides who to integrate with, not you. You can't message Signal friends from WeChat.
+
+**Manual coordination** for everything else—you copy data between tools, maintain consistency, context-switch endlessly.
+
+Your contacts live in one app, dietary preferences in another, calendar events in a third. You are the integration layer.
+
+## The Paradigm Shift
+
+Large Language Models enable anyone to generate specialized tools. A food blogger creates a meal planner. A fitness coach builds a nutrition tracker. Each speaks its own language.
```

Comment:
> Large Language Models enable anyone to generate specialized tools.

This is true, but it would be cool to say:

> Large Language Models enable just-in-time (JIT) integrations, tools and adapters.